test/dataflowtask_test.jl

```diff
@@ -100,13 +100,15 @@ end
     @test_throws TaskFailedException wait(t)
     @test isempty(ERRORS)
 
-    # Errors are handled when debug mode is on
+    # Errors are handled when debug mode is on.
     DataFlowTasks.enable_debug(true)
-    empty!(ERRORS)
-    t = @dspawn error("Expected error")
-    @test_throws TaskFailedException wait(t)
-    @test length(ERRORS) == 1
-    @test ERRORS[1][1].exception.msg == "Expected error"
+    if VERSION >= v"1.7" # FIXME
+        empty!(ERRORS)
+        t = @dspawn error("Expected error")
+        @test_throws TaskFailedException wait(t)
+        @test length(ERRORS) == 1
+        @test ERRORS[1][1].exception.msg == "Expected error"
+    end
```

Comment:
@ffevotte is it difficult to add support for the handling of errors in `v1.6`?

Reply:
My feeling is that we don't really need `errormonitor` at all, and errors could be handled by  [`handle_errors`](https://github.com/maltezfaria/DataFlowTasks.jl/blob/main/src/utils.jl#L50-L62) alone. If that is the case, then test failures with 1.6 are not really related to `errormonitor`, but maybe something else.

But all these are only uneducated guesses at this stage. Let me try and reproduce errors locally; I'll investigate and report back.